bgpd/bgp_evpn.c

```diff
@@ -1914,10 +1903,18 @@ static int update_evpn_route_entry(struct bgp *bgp, struct bgpevpn *vpn,
 			}
 		}
 
-		memcpy(&tmp_pi->extra->label, label, sizeof(label));
-		tmp_pi->extra->num_labels = num_labels;
+		setlabels(&new_attr, &label[0], num_labels);
+
+		/* Add (or update) attribute to hash. */
+		attr_new = bgp_attr_intern(&new_attr);
```

Comment:
because, in previous commit the label was modified just after, but was not impacting the attr structure.
now, because the label has been modified, we have to move the intern of the attr just after the label is calculated.